File: Week4/zad3.js
const me = {
    firstName: 'Stefan',
    lastName: 'Jeftic',
    abbrivatedName: function() {
        return `${this.firstName[0]}. ${this.lastName[0]}.`
    }
}
const date1 = new Date();
console.log(me.abbrivatedName());
date1.setDate(1)
console.log(date1)

File: Python/zad3.py
class Person:
    def __init__(self,firstName,lastName):
        self.firstName = firstName
        self.lastName = lastName

    def abbrivatedName(self):
        print(f"{self.firstName[0]}. {self.lastName[0]}.")


ja = Person("Stefan", "Jeftic")
ja.abbrivatedName()
